### DISCUSSION TITLE
PUT at least code to get the selected item

### DISCUSSION THREAD

**Discussion by Samiakaraeen** [kevlatus/flutter_fortune_wheel#111] [Ideas]
how to get the selected ite , if no code given thi sis usless

**Comment by kevlatus** [comment]
Hi @Samiakaraeen, please refer to the last time when this question has been answered: #66 

You don't get a value from the wheel, instead you tell the wheel which value it is supposed to show. In this example you can see the value is known before it is added to the StreamController. So you can use it from there:

```dart
onFling: () {
    int value = Random().nextInt(fortuneWheelItems.length);
    controller.add(value);
    print(value);
},
```

I'd also encourage to get familiar with the basics of [Dart Streams](https://dart.dev/tutorials/language/streams) as this is fundamental to understand the underlying concepts.
